app/client/components/identity/Form/FormField.tsx

```diff
@@ -64,7 +67,9 @@ export const FormSelectField = <T,>(props: FormSelectProps<T>) => {
 			formikProps={props.formikProps}
 		>
 			<Field component="select" name={name}>
-				<option value="" />
+				<option disabled={firstOptionDisabled} value="">
```

Comment:
What happens if `firstOptionDisabled` is `undefined`? Should we provide a default value ?